libr/anal/reflines.c

```diff
@@ -401,7 +401,7 @@ R_API char* r_anal_reflines_str(void *_core, ut64 addr, int opts) {
 	int l;
 	int dir = 0, wide = opts & R_ANAL_REFLINE_TYPE_WIDE;
 	int pos = -1, max_level = -1;
-	int middle = opts & R_ANAL_REFLINE_TYPE_MIDDLE;
+	int middle_before = opts & R_ANAL_REFLINE_TYPE_MIDDLE_BEFORE;
```

Comment:
all those vars must be bool

Reply:
done in d40178a23fc59d20b52a9b0c9151e2da436a9108